dockers/README.md

```diff
@@ -0,0 +1,227 @@
+Just to note, many of the images maintained in WARP require a handful of system-level dependencies and custom packages so multi-stages builds are typically not used.
+
+##### :eyes: Example
+```dockerfile
+
+# BAD - uses multiple RUN steps
+RUN set eux
+RUN apk add --no-cache curl bash wget
+RUN wget https://www.somezipfile.com/zip
+RUN unzip zip
+
+# GOOD - uses single RUN step
+RUN set eux; \
+        apk add --no-cache \
+            curl \
+            bash \
+    ; \ 
+    wget https://www.somezipfile.com/zip; \
+    unzip zip
+```
+
+### <a name="publicly"></a> Publicly accessible
+
+The pipelines that we maintain in WARP are designed for public use, ideally we would like our docker images to be publicly available as well. This would mean the following conditions must be true.
+
+* Anybody can pull our images
+* Anybody can build our images
+
+For anybody to be able to pull our images they must be hosted on a public container registry, we host all of our images in publics repos on GCR (our 'official' location) and Quay (for discoverability).
+
+* GCR - `us.gcr.io/broad-gotc-prod`
+* Quay - `quay.io/broadinstitute/broad-gotc-prod`
+
+For anybody to be able to build our images all of the functionality should be encapsulated in the Dockerfile. Any custom software packages, dependencies etc. have to be downloaded from public links within the Dockerfile, this obviously means that we should not be copying files from within the Broad network infrastucture into our images.
+
+### <a name="scanning"></a> Image scanning
+
+
+All of the images that we build are scanned for critical vulnerabilities on every pull request. For this we use a github-action that leverages [trivy](https://github.com/aquasecurity/trivy) for scanning. If you build a new image please add it to the action [here](../.github/workflows/trivy.yml).
+
+### <a name="semantic"></a> Semantic tagging
+
+
+We recommend against using rolling tags like `master` or `latest` when building images. Rolling tags make it hard to track down versions of images since the underlying image hash and content could be different across the same tags. Instead we ask that you use a semantic tag that follows the convention below:
+
+##### `us.gcr.io/broad-gotc-prod/samtools:<image-version>-<samtools-version>-<unix-timestamp>` 
+
+This example is for an image we use containing `samtools`. The 'image-version' in this case is the traditional `major.minor.patch` version of the image being built, which is updated when changes to the image (underlying OS, system level packages, etc.) unrelated to `samtools` are made. The 'samtools-version' here correlates with the specific version of `samtools` being used, having this information in the tag makes it easy for users to identify and not have to track down.
```

Comment:
May want to mention that the timestamp is included to address potential issues with Cromwell image caching

Reply:
got it!